visual-js/visual/src/cli/baselines.ts

```diff
@@ -0,0 +1,56 @@
+import { Command, CommandOptions } from 'commander';
+import { VisualApiRegion } from '../regions';
+import { VisualApi, getApi } from '../api';
+
+function apiFromOptions(opts: { region: VisualApiRegion }): VisualApi {
+  return getApi({
+    region: opts.region,
+    user: process.env.SAUCE_USERNAME,
+    key: process.env.SAUCE_ACCESS_KEY,
+  });
+}
+
+type BuildStatusCommandOptions = {
```

Comment:
I guess that should be this:

```suggestion
type BaselineMergeCommandOptions = {
```